zerver/tests/webhooks/test_bitbucket.py

```diff
@@ -46,7 +46,7 @@ def test_bitbucket2_on_fork_event(self):
 
     def test_bitbucket2_on_commit_comment_created_event(self):
         # type: () -> None
-        expected_message = u"User Tomasz(login: kolaszek) added [comment](https://bitbucket.org/kolaszek/repository-name/commits/32c4ea19aa3af10acd08e419e2c354941a365d74#comment-3354963) to commit."
+        expected_message = u"kolaszek [commented](https://bitbucket.org/kolaszek/repository-name/commits/32c4ea19aa3af10acd08e419e2c354941a365d74#comment-3354963) [Commit](https://bitbucket.org/kolaszek/repository-name/commits/32c4ea19aa3af10acd08e419e2c354941a365d74)\n\n~~~ quote\nNice fix!\n~~~"
```

Comment:
not sure if we should change subject here - I keep repo name because subject like:
`repo_name / Commit #longshathatnobodyknows` would be strange. But please let me know if its OK.